### PR TITLE
this is a test

### PR DESCRIPTION
**Pull request by andymckay**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1 @@
-Sorts tabs by URL. Access via accel-shift-o or the tools menu.
-
-Changes:
-
-0.3: added tools menu item.
-     added source to github.
+test 1
```
Comment (andymckay-limited-access) [inline README.md]: sdf

File: lib/main.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// foo bar sdfsdf
```
Comment (andymckay) [inline lib/main.js]: test

File: lib/main.js [hunk 3]
```diff
@@ -1,3 +1,4 @@
+// foo bar asd as sdfsdf
```
Comment (andymckay-limited-access) [inline lib/main.js]: There is a JavaScript syntax error in your code; validation cannot continue on this file.

File: lib/main.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
+// foo bar asd as sdfsdf asd
 var {Hotkey} = require("sdk/hotkeys");
```
Comment (andymckay-limited-access) [inline lib/main.js:2]: There is a JavaScript syntax error in your code; validation cannot continue on this file.

File: lib/main.js [hunk 5]
```diff
@@ -1,3 +1,4 @@
+// asd foo bar asd as sdfsdf asd
 var {Hotkey} = require("sdk/hotkeys");
```
Comment (andymckay-limited-access) [inline lib/main.js:2]: There is a JavaScript syntax error in your code; validation cannot continue on this file.

**Pull request closed without merging**